src/sfctl/helps/main.py

```diff
@@ -104,6 +104,11 @@
     short-summary: Get and delete application resources
 """
 
+helps['mesh deployment'] = """
+    type: group
+    short-summary: Create Service Fabric Mesh resources.
```

Comment:
nit: remove period at the end to be consistent with other help text

Reply:
removed